composer.json

```diff
@@ -34,7 +34,7 @@
         "guzzlehttp/guzzle": "^6.0.0",
         "guzzlehttp/oauth-subscriber": "^0.3.0",
         "jms/serializer-bundle": "^2.3",
-        "knplabs/knp-gaufrette-bundle": "~0.5.2",
+        "knplabs/knp-gaufrette-bundle": "~0.6.1",
```

Comment:
Can you make sure that `~0.5.2` is also still allowed here? Maybe set it to `~0.5` instead